### PR TITLE
Fix linter warnings and fix linter scripts in case of no input files

### PR DESCRIPTION
**Pull request by mikkosest**

### CODE REVIEW

**Comment by nordixinfra** [comment]
Can one of the admins verify this patch?

**Comment by kanwar-saad** [comment]
please sign your commits

File: ci/jobs/linter_tests.pipeline [hunk 1]
```diff
@@ -9,8 +9,14 @@ git_credential_id = "nordixinfra-github-token"
...
   agent { label 'airship-static-workers' }
+  environment {
+      CURRENT_DIR = sh (
+          script: 'pwd',
+          returnStdout: true
+          ).trim()
+  }
   stages {
-    stage('SCM') {
+   stage('SCM') {
```
Comment (maelk) [inline ci/jobs/linter_tests.pipeline]: indentation issue

File: ci/jobs/linter_tests.pipeline [hunk 2]
```diff
@@ -9,8 +9,14 @@ git_credential_id = "nordixinfra-github-token"
 
 pipeline {
   agent { label 'airship-static-workers' }
+  environment {
+      CURRENT_DIR = sh (
```
Comment (maelk) [inline ci/jobs/linter_tests.pipeline]: indentation different from the rest of the file

File: ci/scripts/jumphost/README.md [hunk 3]
```diff
@@ -12,42 +12,43 @@ script
...
 Some environment variables need to be set
 
- - RT_USER: Artifactory username
- - RT_TOKEN: Artifactory token
- - RT_URL: Artifactory URL
+- RT_USER: Artifactory username
+- RT_TOKEN: Artifactory token
+- RT_URL: Artifactory URL
 
+### Key managerment usage
```
Comment (maelk) [inline ci/scripts/jumphost/README.md]: management, typo

File: ci/jobs/linter_tests.pipeline (hunk 2 above)
Comment (mikkosest) [reply ci/jobs/linter_tests.pipeline]: Done

File: ci/scripts/jumphost/README.md (hunk 3 above)
Comment (mikkosest) [reply ci/scripts/jumphost/README.md]: Done

**Pull request merged**